kotlin/tests/src/test/kotlin/com/squareup/moshi/kotlin/codegen/GeneratedAdaptersTest.kt

```diff
@@ -27,6 +27,7 @@ import com.squareup.moshi.Moshi
 import com.squareup.moshi.ToJson
 import com.squareup.moshi.Types
 import com.squareup.moshi.internal.NullSafeJsonAdapter
+import com.squareup.moshi.kotlin.codgen.GeneratedAdaptersTest_MultiplePropertiesShareAdapterJsonAdapter
```

Comment:
Is this right or an auto import from before a recompile?

Reply:
grrr autoimport from IDE

Reply:
795b08b